modules/sdk-coin-eth/src/lib/utils.ts

```diff
@@ -188,10 +189,11 @@ export function flushCoinsData(): string {
 /**
  * Returns the create forwarder method calling data
  *
+ * @param {string} forwarderVersion - forwarder version for the address to be initialized
  * @returns {string} - the createForwarder method encoded
  */
-export function getAddressInitializationData(): string {
-  return createForwarderMethodId;
+export function getAddressInitializationData(forwarderVersion: number): string {
```

Comment:
This parameter is a breaking change. Recommend either making it optional and setting a default, or marking this commit as BREAKING CHANGE in footer. https://www.conventionalcommits.org/en/v1.0.0/

Reply:
made it optional